kvm/v12.3.2/README.md

```diff
@@ -0,0 +1,25 @@
+# :zap: Giant Swarm Release v12.3.2 for KVM :zap:
+
+**Nodes will be rolled during upgrade to this version.**
+
+This patch release prevents an issue with QPS limits introduced by DockerHub.
```

Comment:
```suggestion
This patch release adds registry credentials to prevent an issue with image pulling rate limits recently introduced by Docker Hub.
```